src/data/api/fetch.ts

```diff
@@ -62,6 +62,27 @@ export const sendEventTilVeilarbperson = async (event: FrontendEvent) => {
     return handterRespons(respons);
 };
 
+export const sendOverblikkFilter = async (event: string[]): Promise<string[]> => {
+    const url = `/veilarbfilter/api/overblikkvisning`;
+    const respons = await fetch(url, createPOSToptions(event));
+
+    return handterRespons(respons);
+};
+
+export const useOverblikkFilter = () => {
+    const { data, error, isLoading, mutate } = useSWR<string[], ErrorMessage>(
```

Comment:
The required request-type is a bit different. It needs to be:

```suggestion
    const { data, error, isLoading, mutate } = useSWR<{overblikkVisning: string[]}, ErrorMessage>(
```

See here for reference: https://github.com/navikt/veilarbfilter/blob/dev/src/main/java/no/nav/pto/veilarbfilter/domene/OverblikkVisningRequest.java

Reply:
The same change needs to be made in the `sendOverblikkFilter`-function; the input parameter there should also be `{overblikkVisning: string[]}`.

Reply:
Fixed